src/common/types.cpp

```diff
@@ -267,6 +266,10 @@ bool TypeIsInteger(PhysicalType type) {
 	return (type >= PhysicalType::UINT8 && type <= PhysicalType::INT64) || type == PhysicalType::INT128;
 }
 
+bool TypeIsNested(PhysicalType type) {
```

Comment:
This function appears to be unused, is that intended?

Reply:
Ah, not that is not intended. I used that shortly during development, but in the end it was more useful to check the physical type directly. I will remove it for now.